src/main.js

```diff
@@ -725,19 +741,52 @@ class Wavelet {
      */
     async sendTransaction(wallet, tag, payload, opts = {}) {
         const payload_hex = Buffer.from(payload).toString("hex");
-
+        const sender = Buffer.from(wallet.publicKey).toString("hex");
+        const now = Date.now();
+        const nonceCacheExpire = 3 * 60 * 1000;     // expire nonce cache after 3 mins
```

Comment:
Is there a reason why we expire after 3 minutes explicitly?

Reply:
In order to avoid repeatedly calling nonce for each transfer, we talked about storing the nonce on the client for a while and increment it each time it's used. After it expires it should try to update with the proper value. This might be a temporary solution until we have a WebSocket to get updated nonces.
I chose 3 minutes but I'm happy to change to whatever value is more appropriate.

Reply:
I don't know a more appropriate value, was just wondering why you landed on 3 minutes - I have no objection other than general distrust of hardcoded values. If the wavelet team thinks it's a reasonable number lets go with it